build_tools/ci/print_ir_aie2xclbin/print_ir_aie2xclbin.sh

```diff
@@ -131,7 +131,6 @@ ${SOURCE_MLIR_FILE} \
 --aie2xclbin-print-ir-after-all \
```

Comment:
I suspect that this flag is not actually doing anything: as the flag `--mlir-print-ir-after-all` is also passed, the printing is happening still.

Reply:
I did try this and the printing for the aie2xclbin passes didnt happen without the flag.